Sources/iPad/View Controllers/File/DocumentsViewController_iPad.m

```diff
@@ -175,6 +175,9 @@ - (void)buttonAccessoryClick:(id)sender{
     [_actionPopoverController release];
 	_actionPopoverController = [[WEPopoverController alloc] initWithContentViewController:fileActionsViewController];
     _actionPopoverController.delegate = self;
+    // set Properties (Border in black gradient)
+    [_actionPopoverController setContainerViewProperties:[self improvedContainerViewProperties]];
```

Comment:
If you use a `@property` as explained above, use the dot-notation here: `self.improvedContainerViewProperties`